src/Diagnostics.DataProviders/DataProviders/SupportObserverDataProvider.cs

```diff
@@ -200,7 +240,7 @@ public override HttpClient GetObserverClient()
         private async Task<string> Get(string path)
         {
             var request = new HttpRequestMessage(HttpMethod.Get, $"https://support-bay-api.azurewebsites.net/observer/{path}?api-version=2.0");
```

Comment:
In appsettings.json we have a Support Observer endpoint, how this is one different from that?

Reply:
I can talk more about this with you offline but we have two services that we rely on and this is one of them